content/courses/Ada_For_The_C_Embedded_Developer/chapters/02_Perspective.rst

```diff
@@ -1180,9 +1180,65 @@ separate representation clause for it like:
 
 Note that however, unlike C, values for enumerations in Ada have to be unique.
 
-.. todo::
+Unsigned And Modular Types
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Unsigned integer numbers are quite common in embedded applications. In C, you
+can use them by declaring :c:`unsigned int` variables. In Ada, you can use
+:ada:`Natural` variables, which is a subtype of :ada:`Integer`. For example:
```

Comment:
This is a little confusing because you are comparing a C unsigned type with an Ada signed type without clarifying that Natural is indeed signed, with an explicit range attached to the subtype.